circle.yml

```diff
@@ -3,20 +3,20 @@ machine:
     LANG: en_US.UTF-8
     LC_CTYPE: en_US.UTF-8
   xcode:
-    version: "6.2"
+    version: "6.3.1"
 dependencies:
   pre:
-    - xcrun instruments -w "iPhone 6 (8.2 Simulator)" || exit 0
+    - xcrun instruments -w "iPhone 6 (8.3 Simulator)" || exit 0
   override:    
-    - sudo gem install cocoapods xcpretty obcd slather -N
-    - pod install --project-directory=Example
+    - bundle check --path=vendor/bundle || bundle install --path=vendor/bundle --jobs=4
```

Comment:
`bundle install` should never hit the network if everything in the lockfile is available?